### PR TITLE
docs: show example using base-url

### PR DESCRIPTION
**Pull request by eh-am**

### CODE REVIEW

**Comment by Rperry2174** [comment]
Where do you think we might be able to add this to the docs repo @eh-am @petethepig ?

**Review by eh-am — COMMENTED** [review]
Just noticed we don't need to expose the pyroscope server use host network.

File: examples/base-url/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    image: 'pyroscope/pyroscope:latest'
+    ports:
+      - '4040:4040'
+    command:
+      - 'server'
+    environment:
+      PYROSCOPE_BASE_URL: /pyroscope
+  nginx:
+    image: nginx
+    network_mode: host
```
Comment (petethepig) [inline examples/base-url/docker-compose.yml:13]: this won't work on mac (and windows too I think)

it's more portable to refer to hosts using the hostnames:
```
proxy_pass http://pyroscope:4040;
```

and then you don't need `network_mode: host` either

Comment (eh-am) [reply examples/base-url/docker-compose.yml]: Yeah I was using that setup to test this base url stuff against a local pyroscope instance, and forgot to update that.

**Pull request merged**